### ISSUE TITLE
twitter-to-sqlite user-timeline [screen_names] --sql / --attach

### ISSUE THREAD

**Issue by simonw** [dogsheep/twitter-to-sqlite#35]
Split from #8.

**Comment by simonw** [comment]
I'm going to redesign the command a little. Right now it defaults to fetching the authenticated user's timeline and allows `--screen_name` to be used to fetch for one other user instead.

I think the design of `users-lookup` is much better. I'm going to copy that:

```
twitter-to-sqlite users-timeline users.db simonw cleopaws
twitter-to-sqlite users-timeline users.db 12497 3166449535 --ids
```
It will also accept `--since` and `--sql` and `--alter` and `--since_id`.

**Comment by simonw** [comment]
I can give it a snazzier progress bar to, as requested by #10.

**Comment by simonw** [comment]
I'm going to keep `--screen_name` working for backwards compatibility. Using `hidden=True` on it should hide it from the help output.

**Comment by simonw** [comment]
Needs documentation.

**Comment by simonw** [comment]
Released as 0.18.
